src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -126,29 +126,29 @@ <h4 ng-if="$ctrl.useV3 === 'true'" class="panel-title  border-bottom-small  visi
               <label class="mb0 hidden-xs hidden-sm">
                 <span class="invisible mb0"></span>
               </label>
-              <button type="button" class="btn btn-primary btn-block" ng-click="$ctrl.toggleSpouseDetails()" uib-tooltip="{{'By adding your spouse\'s name, we can ensure that both of your contributions are linked together, providing a clearer and more accurate record of your household\'s giving.' | translate}}">
-                  <span ng-if="!$ctrl.showSpouseDetails">Add Spouse</span>
-                  <span ng-if="$ctrl.showSpouseDetails">Remove Spouse</span>
+              <button type="button" class="btn btn-primary btn-block" ng-click="$ctrl.toggleSpouseDetails()"
+                uib-tooltip="{{'By adding your spouse\'s name, we can ensure that both of your contributions are linked together, providing a clearer and more accurate record of your household\'s giving.' | translate}}">
```

Comment:
If you want this on a separate line, line it up with `type`, and also put `class` and `ng-click` on their own lines and line them all up.